frontend/src/PlayerComponents/NewPlayerModal.tsx

```diff
@@ -20,12 +20,13 @@ type PropsPlayerModal = {
     visibilitySaveToChangePlayerButton: boolean
     visibilityDeletePlayerButton: boolean
     allPlayersList: () => void
+    user:string
 
 }
 export default function NewPlayerModal(propsPlayerModal: PropsPlayerModal) {
-    const [firstName, setFirstName] = useState(propsPlayerModal.player?.firstName || "")
-    const [lastName, setLastName] = useState(propsPlayerModal.player?.lastName || "")
-    const [age, setAge] = useState(propsPlayerModal.player?.age || 0)
+    const [firstName, setFirstName] = useState(propsPlayerModal.player?.firstName ?? "")
```

Comment:
```suggestion
    const [firstName, setFirstName] = useState(propsPlayerModal.player.firstName ?? "")
```
optional chaining brauchst du eigentlich nicht, wenn du sowieso ?? benutzt.

Reply:
Die Fehlermeldung bei deinem Vorschlag:  propsPlayerModal.player  is possibly  undefined 

Reply:
okay gut, dann könnte es wohl doch nen fehler werfen, weil es nicht an den firstName kommt. na gut :D